js/examples/read_file.html

```diff
@@ -41,7 +41,7 @@
 }
 reader.onload = function (evt) {
 
-  var arrowTable = Arrow.Table.from([new Uint8Array(evt.target.result)]);
+  var arrowTable = Arrow.tableFromIPC([new Uint8Array(evt.target.result)]);
```

Comment:
Does this work? 

```suggestion
  var arrowTable = Arrow.tableFromIPC(evt.target!.result);
```

Reply:
The suggested revision `Arrow.tableFromIPC(evt.target!.result)` does not work.  Console error: "Uncaught SyntaxError: missing ) after argument list".

Reply:
If we change it to:

```js
var arrowTable = Arrow.tableFromIPC(evt.target.result);
```

it will work.

Reply:
Let's use that then.

Reply:
OK, done.